### DISCUSSION TITLE
Add any track in multiple playlists

### DISCUSSION THREAD

**Discussion by zappere50** [pear-devs/pear-desktop#728] [Ideas]
Sometimes I need to add some track to a couple of playlists, but for that I need to click Add to playlist again and I would like to select the playlists where I want to add this, like on YouTube site
![Screenshot_2](https://user-images.githubusercontent.com/83653958/169906538-efe58cdb-d722-4bf9-a93c-0567e889ea9c.png)

**Comment by r1yaz** [comment]
+1 to this feature. Is there any way to accomplish this?
